pkg/kp/rcstore/store.go

```diff
@@ -46,3 +52,33 @@ type Store interface {
 	// The two output channels will be closed in response.
 	Watch(rc *fields.RC, quit <-chan struct{}) (<-chan struct{}, <-chan error)
 }
+
+func RCPath(rcId fields.ID) (string, error) {
```

Comment:
Unless I missed something, this doesn't need to be public.  And it runs against your goals to have a public function that returns a string path.